guides/common/modules/proc_synchronizing-the-satellite-tools-repository.adoc

```diff
@@ -1,26 +1,87 @@
 This repository provides the `katello-agent`, `katello-host-tools`, and `puppet` packages for clients registered to {ProjectServer}.
-endif::[]
+Select the applicable operating system and version to synchronize the client repository.
+
+* xref:#synchronizing-repos-rhel9-rhel8[{RHEL} 9 & {RHEL} 8]
+* xref:#synchronizing-repos-rhel7-rhel6[{RHEL} 7 & {RHEL} 6]
+
+== [[synchronizing-repos-rhel9-rhel8]]{RHEL} 9 & {RHEL} 8
+
+To use the CLI instead of the {ProjectWebUI}, see the xref:CLI_Synchronizing_the_Client_Repository_rhel_9_{context}[] or xref:CLI_Synchronizing_the_Client_Repository_rhel_8_{context}[].
 
 .Procedure
 . In the {ProjectWebUI}, navigate to *Content* > *Sync Status*.
+. Click the arrow next to the *{RHEL} for x86_64* product to view available content.
+. Select *{Team} {project-client-name} for RHEL 9 x86_64 RPMs* or *{Team} {project-client-name} for RHEL 8 x86_64 RPMs* whichever is applicable.
+. Click *Synchronize Now*.
+
+[id="CLI_Synchronizing_the_Client_Repository_rhel_9_{context}"]
+.CLI procedure for {RHEL} 9
+* Synchronize your {project-client-name} repository using the `hammer repository synchronize` command:
 +
-A list of product repositories available for synchronization is displayed.
-. Click the arrow next to the *{RHEL} Server* product to view available content.
-. Select *{project-client-name} (for RHEL 8) RPMs x86_64*.
+[options="nowrap" subs="+quotes,attributes"]
+----
+# hammer repository synchronize \
+--name "{Team} {project-client-name} for RHEL 9 x86_64 RPMs" \
+--organization _"My_Organization"_ \
+--product "{RHEL} for x86_64"
+----
+
+[id="CLI_Synchronizing_the_Client_Repository_rhel_8_{context}"]
+.CLI procedure for {RHEL} 8
+* Synchronize your {project-client-name} repository using the `hammer repository synchronize` command:
++
+[options="nowrap" subs="+quotes,attributes"]
+----
+# hammer repository synchronize \
+--name "{Team} {project-client-name} for RHEL 8 x86_64 RPMs" \
+--organization _"My_Organization"_ \
+--product "{RHEL} for x86_64"
+----
+
+== [[synchronizing-repos-rhel7-rhel6]]{RHEL} 7 & {RHEL} 6
+
+[NOTE]
+====
+You will require *{RHEL} Extended Life Cycle Support (ELS) Add-on*
+to synchronize the repositories in the older versions of RHEL.
+For more information,
+see https://www.redhat.com/en/resources/els-datasheet[Extended Life Cycle Support (ELS) Add-on] guide for {RHEL}.
+====
+
+To use the CLI instead of the {ProjectWebUI}, see the xref:CLI_Synchronizing_the_Client_Repository_rhel_7_{context}[] or xref:CLI_Synchronizing_the_Client_Repository_rhel_6_{context}[].
+
+.Procedure
+. In the {ProjectWebUI}, navigate to *Content* > *Sync Status*.
+. Click the arrow next to the *{RHEL} Server* or *{RHEL} Server - Extended Life Cycle Support* whichever product is applicable to view available content.
+. Select *{Team} {project-client-name} (for RHEL 7 Server) RPMs x86_64* or *{Team} {project-client-name} for RHEL 6 Server - ELS RPMs x86_64* whichever is applicable.
```

Comment:
"whichever is available" sounds a bit weird to me.

Reply:
Addressed with better wordings :+1: